cmake/Modules/uninstall.cmake

```diff
@@ -0,0 +1,80 @@
+set(MANIFEST "${CMAKE_BINARY_DIR}/install_manifest.txt")
+
+if(NOT EXISTS ${MANIFEST})
+    message(FATAL_ERROR "Cannot find install manifest: '${MANIFEST}'")
+endif()
+
+file(STRINGS ${MANIFEST} files)
```

Comment:
would it be easier to add all files into a single `files` variable and iterate over it once?

Reply:
done